src/Application/Products/DTOs/ProductDto.cs

```diff
@@ -0,0 +1,3 @@
+﻿namespace Application.Products.DTOs;
```

Comment:
There is `DTOs` in one namespace but `Api` in another. It would be nice if we consolidated capitalisation :) If we vote, then I vote for `API` in the namespace :)

Reply:
I've gone for using `Dto` for now
We could open a discussion, though...

Reply:
I think this one is important, not only for the naming, but also when using docker. Beware that there can be some problems because of the renamings when using Docker on Linux vs the Windows hold code. So, the sooner we get an agreement for this one, the better :)